charts/team-ns/templates/tekton-tasks/grype.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $v := .Values }}
```

Comment:
Does not seem to be needed

Reply:
indeed. I thought they where already there.